src/components/SelectBoxContainer.tsx

```diff
@@ -0,0 +1,171 @@
+import { useState, useEffect } from "react";
+import Dropdown from "./Dropdown";
+import bookCodesData from "../assets/data/book_codes.json";
+import versificationData from "../assets/data/versification.json";
+import { VersificationData } from "../types/bible";
+import { BookOption, ChapterOption, VerseOption, OptionType } from "../types/Navigation"
+
+const SelectBoxContainer = () => {
+  const [selectedBook, setSelectedBook] = useState<BookOption | null>(null);
```

Comment:
Book, chapter, verse can be moved to a store later on like zustand or react context to be used by other components.